Cargo.toml

```diff
@@ -194,3 +194,6 @@ strip = true
 #
 # [patch."https://github.com/servo/<repository>"]
 # <crate> = { path = "/path/to/local/checkout" }
+
+# patch dwrote for https://github.com/servo/dwrote-rs/pull/55
+dwrote = { git = "https://github.com/servo/dwrote-rs", rev = "f2f644d3de8dee9a71c90e0376fd44531e5c2eec" }
```

Comment:
I think you can just specify branch and it will take the latest commit

Reply:
dwrote v0.11.1 is now published.